koku/api/migrations/0038_create_hive_db.py

```diff
@@ -0,0 +1,104 @@
+# Generated by Django 3.1.7 on 2021-03-03 19:26
+import logging
+import os
+
+from django.db import migrations
+
+
+LOG = logging.getLogger(__name__)
+
+
+def create_hive_db(apps, schema_editor):
+    rolname = os.environ.get("HIVE_DATABASE_USER", "hive")
+    datname = os.environ.get("HIVE_DATABASE_NAME", "hivedb")
```

Comment:
The db name in ci/qa is `hive`

Reply:
We'll have to make sure the koku deployment gets these env vars if we're to run this as-is, so I'd almost rather make certain they're there before we merge this.

Reply:
:+1: